src/EOS/Transaction.cpp

```diff
@@ -99,14 +100,19 @@ void Transaction::serialize(Data& os) const noexcept{
     encodeCollection(transactionExtensions, os);
 }
 
-json Transaction::serialize() const {
+std::string Transaction::formatDate(int32_t date) {
+    // format is "2022-06-02T08:53:20", always 19 characters long
+    constexpr size_t DateSize = 19;
+    constexpr size_t BufferSize = DateSize + 1;
+    char formattedDate[BufferSize];
```

Comment:
I think using a string stream or an std::array will make more sense here, if you use stringstream, you can use std::put_time to fill the buffer